src/components/Card/index.tsx

```diff
@@ -342,6 +358,13 @@ const Card: React.FC<CardPropTypes> = ({
           {leftAdornment && <LeftAdornment>{leftAdornment}</LeftAdornment>}
 
           <TextContent hasImage={hasImage}>
+            {period && (
+              <Period>
+                <CalendarAlt size={16} color={color.mars} />
+                <PeriodText color={color.mars}>{period}</PeriodText>
```

Comment:
Maybe a question for Mark: Is the text supposed to be red? I think the iOS app is not red text.

Reply:
Yeah, on the iOS app we have non-bold titles and in that case the red drowns out the title. On Web that's less of an issue and the red gives it some more graphical swung. It can go either way, but the difference is intended.

Reply:
I now have the possibility to custom the color of the header, it would default to Tiny Text color if not precised